Makefile

```diff
@@ -145,3 +147,12 @@ help:					#: Show the help messages
 	@cat $(firstword $(MAKEFILE_LIST)) | \
 	awk '/^[-a-z]+:/' | \
 	awk -F: '{ printf("%-20s   %s\n", $$1, $$NF) }'
+
+python-clean:
+	DEBUG=1 ${PYTHON_EXECUTABLE} setup.py clean
+
+python-build:
+	DEBUG=1 ${PYTHON_EXECUTABLE} setup.py develop
```

Comment:
so always build with `DEBUG=1`?

Reply:
Atleast for now , I will remove in future.